applications/nrf_desktop/README.rst

```diff
@@ -7,7 +7,7 @@ nRF Desktop
    :local:
    :depth: 2
 
-The nRF Desktop is a reference design of a Human Interface Device (HID) that is connected to a host through Bluetooth LE or USB, or both.
+The nRF Desktop is a reference design of a Human Interface Device (HID) that is connected to a host through Bluetooth® Low Energy or USB, or both.
```

Comment:
Any reason why you choose to expand the LE abbreviation here but not other places?

Reply:
Mainly the fact that HID was also expanded in the same sentence, so it felt natural to expand LE as well.